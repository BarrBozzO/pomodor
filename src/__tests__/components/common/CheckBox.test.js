import React from "react";
import renderer from "react-test-renderer";

import Checkbox from "../../../components/common/CheckBox";

it("should render Button component correctly", () => {
  const id = "testId",
    checked = false,
    disabled = true,
    name = "testName",
    handleChange = jest.fn();

  const tree = renderer
    .create(
      <Checkbox
        id={id}
        checked={checked}
        disabled={disabled}
        handleChange={handleChange}
        name={name}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
