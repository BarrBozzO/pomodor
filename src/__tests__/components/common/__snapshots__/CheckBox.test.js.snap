// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button component correctly 1`] = `
<div
  className="sc-bdVaJa bFnqau"
>
  <input
    checked={false}
    className="sc-bwzfXH iqDort"
    disabled={true}
    id="testId"
    onChange={[MockFunction]}
    type="checkbox"
  />
  <label
    htmlFor="testId"
  />
</div>
`;
